Core/Core/Dashboard/K5/View/Schedule/K5ScheduleEntryView.swift

```diff
@@ -153,7 +153,7 @@ public struct K5ScheduleEntryView: View {
                 background.overlay(icon)
             }
         })
-        .accessibility(label: Text("Mark item as done", bundle: .core))
+        .accessibility(label: Text(viewModel.title) + Text(",") + Text("Mark item as done", bundle: .core))
```

Comment:
We don't want that comma to get translated so let's use `Text(verbatim: ",")` instead.